now.json

```diff
@@ -1,10 +1,26 @@
 {
-  "version": 1,
-  "type": "docker",
+  "version": 2,
   "alias": [
-    "badgen.net",
-    "flat.badgen.net"
+    "v2.badgen.net"
   ],
+  "regions": ["sfo", "lhr", "hnd"],
+  "builds": [
```

Comment:
You can delete the `builds` property and make sure your package.json `build` script outputs a `public` directory.

Reply:
@styfle Next.js doesn't allow using `public` for build output 🤔

https://github.com/zeit/next.js/blob/master/errors/can-not-output-to-public.md

Reply:
Oh, I didn't realize you're using Next.js since you didn't use `@now/next`.

In that case, just follow Next.js documentation, no need `builds` key or really any build scripts in `package.json`

Reply:
Prefer to keep it explicit :D